client-common/src/tendermint/lite.rs

```diff
@@ -8,7 +8,7 @@ use crate::Result as CommonResult;
 
 ///
 #[derive(Clone, Debug, Deserialize, Serialize)]
-pub struct TrustedState(pub(crate) Option<lite::TrustedState<SignedHeader, Header>>);
+pub struct TrustedState(pub Option<lite::TrustedState<SignedHeader, Header>>);
```

Comment:
why not pub(crate)?

Reply:
Oh, that just used to get the `Option<...>` to debug, I'll rollback.